### ISSUE TITLE
Strange culture conversions

### ISSUE THREAD

**Issue by RedArkady** [MattTheLegoman/RealmsInExile#144]
People keep converting to uruk culture.

In general I'm not sure whether those of Numenorian heritage ought to be converting to local culture, uruk or not.

**Comment by RedArkady** [comment]
![Crusader Kings III Screenshot 2021 06 27 - 14 27 39 21](https://user-images.githubusercontent.com/20355423/123546331-14bbaa80-d754-11eb-96fc-aabda137b813.png)

**Comment by Remerod** [comment]
So I tried out to see if this is fixed!

I runned a game on observer for 50 years and scrolled threw all of the uruk culture groups in find characters search engine, i found no "human" looking orcs

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/146571107-7a58caf1-557f-4e51-b023-1ca7ff6cc6f7.png)

</details>

So i continued from 50 years but tweaked a bit, i "disposed" Sauron and made it so a Núrniag took control over mordor, to make sure to simulate a human culture ruling over many orcs cultures. i continued to run 50 more years so 100 years in all. I searched for all uruk cultures and i could not find any human to orc conversions.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/146579380-887eef88-ad27-498f-9355-903ef63c9161.png)

</details>

So while not directly testing the a numenoreans converting to uruk culture, i did check around if any "numenorean" culture had converting, and all i could find was a umberan converting to tedjin culture in phurut-Azrazâin (in a fast 2min search)

So i don't think this is a problem any more, but i could run a game where i land a numenorean in mordor if more testing is wished for.
